sabnzbd/par2file.py

```diff
@@ -93,13 +93,17 @@ def parse_par2_file(fname: str, md5of16k: Dict[bytes, str]) -> Dict[str, bytes]:
     """
     table = {}
     duplicates16k = []
+    seenfiles = []
 
     try:
         with open(fname, "rb") as f:
             header = f.read(8)
             while header:
                 name, filehash, hash16k = parse_par2_file_packet(f, header)
                 if name:
+                    if name in seenfiles:
```

Comment:
Good catch, indeed we can assume all `FileDesc` is unique.
But I don't think we need `seenfiles`, why not just use `name in table`?